docs/io.cozy.apps.md

```diff
@@ -74,3 +74,15 @@ This is the truncated version of the official Cozy `drive` app.
   }
 }
 ```
+
+# Specials `io.cozy.apps` doctypes 
+
+We have a few `magics folders` in the cozy ecosytem like `Admistrative` or `Photos`... In order to let the user renames these folders without having our app re-creating them everytime, we have created a few specials doctypes to use them in the [`referenced_by` attribute](https://docs.cozy.io/en/cozy-stack/references-docs-in-vfs/#post-filesfile-idrelationshipsreferenced_by) . 
+
+Here is the list: 
+```
+ADMINISTRATIVE FOLDER: io.cozy.apps/administrative
+PHOTOS FOLDER: io.cozy.apps/photos
+PHOTOS_BACKUP FOLDER: io.cozy.apps/photos/mobile
+PHOTOS_UPLOAD FOLDER: io.cozy.apps/photos/upload
```

Comment:
Can you add a description ? The difference between the photos folders is particularly unclear to me.